tests/web/mock_web_api_server.py

```diff
@@ -111,7 +111,7 @@ def _handle(self):
                     return
                 if pattern == "rate_limited":
                     self.send_response(429)
-                    self.send_header("Retry-After", 30)
+                    self.send_header("Retry-After", 1)
```

Comment:
Changed the value for faster test execution (we really don't want to wait for 30 seconds